frontend/public/components/factory/table.tsx

```diff
@@ -378,7 +378,7 @@ export const Table = connect<TablePropsFromState,TablePropsFromDispatch,TablePro
       }
 
       // re-render after resize
-      window.addEventListener('resize', this._handleResize);
+      window.addEventListener('resize', _.debounce(this._handleResize, 100));
```

Comment:
Does `removeEventListener` work with this change? We might want to specify `_.debounce` above in the constructor.

Reply:
good one @spadgett 👍 . updated.